Ponder rework

The-Shortman opened the pull request:
This pull request:

- [x] Standardises the ponder tag registry into a const/forEach loop.
- [ ] Adds a ponder entry for all custom machinery currently missing a ponder.
- [ ] Reworks existing ponder entries to match the style of the rest of the ponder index in terms of grammar and legibility.
- [ ] Improves the animations of the existing ponders to give them a bit more of a visual flair.
- [ ] Adds lang keys to the ponders for i18n purposes.

The-Shortman committed:
New ponder tag registry template

nageih commented on kubejs/client_scripts/ponder_misc.js:
```diff
@@ -0,0 +1,86 @@
+        namespace: "kubejs:createastral_tips", // Anything unique to astral that isn't a machine or part of a kubejs addon e.g. astralsignals
+        icon: "createastral:astral_singularity",
+        displayName: "Create: Astral Tips",
+        tooltip: "Tips and tricks specific to this modpack",
```
Just a reminder, I'm not sure how it works. If necessary, the texts may need to be translated. Thank you for your help.

nageih replied:
Perhaps it will be automatically generated in `kubejs/assets/ponderjs_generated/lang`? I'm not sure.

The-Shortman replied:
I'll add this to the to-do list

The-Shortman committed:
Update ponder types to work with namespaces

The-Shortman committed:
Remove old ponders (dw i have backups for reference)

The-Shortman committed:
Implement new desizer ponders

The-Shortman committed:
Catalogue some blocks into the new ponder tags

The-Shortman committed:
Add shimmer refinery ponder

The-Shortman committed:
Merge remote-tracking branch 'origin/Astral-Experimental' into feature/ponders

The-Shortman committed:
Merge remote-tracking branch 'origin/Astral-Experimental' into feature/ponders

The-Shortman committed:
Rename Void Filter to Shimmer Refinery Dummy
Did this so that the ponder index shows 'shimmer refinery' instead of 'void filter' to reduce confusion

The-Shortman committed:
Merge remote-tracking branch 'origin/Astral-Experimental' into feature/ponders

The-Shortman committed:
Add first shimmer ponder
Second one is planned to showcase transmutation
